src/Microsoft.Security.Utilities.Core/PreciselyClassifiedSecurityKeys/AzureCognitiveServices.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Security.Utilities;
+
+public enum AzureCognitiveServices
+{
+    AzureAIServices = 0,
+    AzureOpenAI = 1,
+    AzureAnomalyDetectorEE = 2,
+    AzureAnomalyDetector = 3,
+    AzureCognitiveServices = 4,
+    AzureComputerVision = 5,
+    AzureContentModerator = 6,
+    AzureContentSafety = 7,
+    AzureCustomVisionPrediction = 8,
+    AzureCustomVisionTraining = 9,
+    AzureFace = 10,
+    AzureFormRecognizer = 11,
+    AzureHealthDecisionSupport = 12,
+    AzureHealthInsights = 13,
+    AzureImmersiveReader = 14,
+    AzureInternalAllInOne = 15,
+    AzureKnowledge = 16,
+    AzureLuisAuthoring = 17,
+    AzureLuis = 18,
+    AzureMetricsAdvisor = 19,
+    AzurePersonalizer = 20,
+    AzureQnAMaker = 21,
+    AzureQnAMakerv2 = 22,
```

Comment:
Since there are others like this, I assume it's intentional, but should the casing be `V2` with a capital V?

Reply:
Yes, this casing mistake was picked up from provider project, there's no problem correcting it, done.